integration/tests/neon_evm/test_compute_units.py

```diff
@@ -0,0 +1,563 @@
+            operator=deterministic_operator_keypair,
+            user=deterministic_sender_with_tokens,
+            contract_file_name="common/Common",
+            neon_api_client=neon_api_client,
+            treasury_pool=deterministic_treasury_pool,
+            version="0.8.12",
+        )
+
+        nonce = evm_loader.get_neon_nonce(
+            account=deterministic_neon_user.neon_address,
+            chain_id=evm_loader.sol_chain_id,
+        )
+        contract_data = 18
+        data = decode_function_signature(
+            function_name="setNumber(uint256)",
+            args=[contract_data],
+        )
+        tx = ScheduledTransaction(
+            payer=deterministic_neon_user.neon_address,
+            sender=None,
+            nonce=nonce,
+            index=0,
+            target=basic_contract.eth_address,
+            value=0,
+            call_data=data,
+            chain_id=evm_loader.sol_chain_id,
+        )
+        tree_account = evm_loader.create_tree_account(
+            neon_user=deterministic_neon_user,
+            treasury=deterministic_treasury_pool,
+            transaction=tx.encode(),
+        )
+        transaction_tree_data = neon_api_client.get_transaction_tree(
+            address=deterministic_neon_user.neon_address.hex(),
+            nonce=nonce,
+        )
+        assert transaction_tree_data.get_transaction_count() == 1
+
+        evm_loader.write_transaction_to_holder_account(
+            tx.encode(), deterministic_holder_acc, deterministic_operator_keypair
+        )
+        additional_accounts = [
+            basic_contract.solana_address,
+            deterministic_neon_user.get_balance_account(evm_loader.sol_chain_id),
+        ]
+
+        evm_loader.start_scheduled_trx_from_account(
+            index=0,
+            operator=deterministic_operator_keypair,
+            holder=deterministic_holder_acc,
+            tree_account=tree_account,
+            additional_accounts=additional_accounts,
+            chain_id=evm_loader.sol_chain_id,
+        )
+
+        operator_balance_pubkey = evm_loader.get_operator_balance_pubkey(
+            operator=deterministic_operator_keypair,
+            chain_id=evm_loader.sol_chain_id,
+        )
+        cu_expected_list = [28300, 29284]
+
+        for i, cu_expected in enumerate(cu_expected_list):
+            receipt = evm_loader.send_transaction_step_from_account(
+                operator=deterministic_operator_keypair,
+                operator_balance_pubkey=operator_balance_pubkey,
+                treasury=deterministic_treasury_pool,
+                storage_account=deterministic_holder_acc,
+                additional_accounts=additional_accounts,
+                steps_count=EVM_STEPS,
+                signer=deterministic_operator_keypair,
+                compute_unit_price=3929,
+            )
+
+            if receipt.value.transaction.meta.err:
+                raise AssertionError(f"Error in sol trx: {receipt}")
```

Comment:
looks like the test is negaitve
Can we add at least 1 positive test for scheduled trx?

Reply:
maybe it's not negative but looks strange that it's only 2 iteration (min number = 3)